### PR TITLE
Introduce Multi_Annotator class

### LINKED ISSUE #31: How are we gonna handle multiple annotators ?
I started writing the loader of `sonyc_ust` which has multiple annotators for each file, and binary labels for a set of classes. I was wondering how do we wanna handle this. I have some ideas but I'm happy to hear what others think:

For the binary labels, since the dataset has a defined taxonomy, I'm gonna get the vector `[0, 1, 0, ... ,1]` corresponding to activations of the different classes `[class_1, class_2, ... , class_N]` from each clip into tags, so the clip will have a list of only the classes present in that clip (e.g. `[class_2, class_N]`. This allows to explore clip-wise metadata easy. For exploring the big picture of the dataset, I refer to #22.

For handling multiple annotators, I see two possible solutions:

Right now `clips.tags = annotations.Tags` which internally has a list of labels and a list of confidence values. 

1) One option is to return a list of `annotations.Tags` and have a separated property for the annotator IDs, ordered in the same way:

```
clip.tags = [annotations.Tags, annotations.Tags, ... ]
clip.annotators = [annotator_1, annotator_2, ... ]  # these are str
```
Good: the datasets will always have the same property `tags`. Bad: `tags` is used in different ways in different datasets and is not so clear the link between annotators and their annotations.

2) We introduce a `muti_annotator` annotation type or property with `tuples` matching annotator ID and annotation:

```
clip.multi_annotator = [(annotation, annotator_ID), (annotation, annotator_ID), ...]
```
In this case `annotation` is `annotations.Tags`.

Good: it's very clear the dataset has a ground truth with multiple annotators. Bad: the annotation type gets a bit buried down in the code, so we have to be super clear in the docs or try to make it very intuitive somehow.

Thoughts? @justinsalamon @mrocamora @pzinemanas @mcartwright

### LINKED ISSUE #48: Remove pretty_midi from setup
Good catch @mrocamora. It was inherited from `mirdata`, it shouldn't be there.

### PR DESCRIPTION
**Pull request by magdalenafuentes**
This PR introduces a `Multi_Annotator` class to handle multiple annotators in a dataset. Closes #31. Fixes #48

### CODE REVIEW

File: soundata/annotations.py [hunk 1]
```diff
@@ -53,259 +53,38 @@ def __init__(self, intervals, labels, confidence=None) -> None:
         self.confidence = confidence
 
 
-class BeatData(Annotation):
-    """BeatData class
+class Multi_Annotator(Annotation):
```
Comment (justinsalamon) [inline soundata/annotations.py]: How about `MultiAnnotation` instead? There may be multiple annotations from the same annotator, so I feel like `MultiAnnotation` is a little broader. Also I suggest we remove the "_" for the class name

File: soundata/annotations.py [hunk 2]
```diff
@@ -53,259 +53,38 @@ def __init__(self, intervals, labels, confidence=None) -> None:
...
+    """Multiple annotator class.
```
Comment (justinsalamon) [inline soundata/annotations.py:57]: Multiple annotation class

**Review by justinsalamon — COMMENTED** [review]
@magdalenafuentes looks great, I made some minor suggestions for nomenclature

File: soundata/annotations.py (hunk 1 above)
Comment (magdalenafuentes) [reply soundata/annotations.py]: Yeah, let's drop the `_`. About the `MultiAnnotation`, I worry that could be interpreted as the clip having multiple annotations (e.g. tags, events), and that is independent of this class. What do you think?

Comment (justinsalamon) [reply soundata/annotations.py]: That's a good point, it could be confused with multi-label. OK, let's go with `MultiAnnotator`

**Comment by magdalenafuentes** [comment]
Also @pzinemanas take a look if you can in case this annotation type makes sense for any of the datasets you were planning to add

File: tests/test_annotations.py [hunk 3]
```diff
@@ -57,160 +59,33 @@ def test_events():
...
-def test_tempo_data():
-    intervals = np.array([[1.0, 2.0], [1.5, 3.0], [2.0, 3.0]])
-    value = np.array([140.0, 110.0, 111.0])
-    confidence = np.array([0.1, 0.4, 0.2])
-    tempo_data = annotations.TempoData(intervals, value, confidence)
-    assert np.allclose(tempo_data.intervals, intervals)
-    assert np.allclose(tempo_data.value, value)
-    assert np.allclose(tempo_data.confidence, confidence)
-
+def test_multiannotators():
```
Comment (justinsalamon) [inline tests/test_annotations.py]: nit: `test_multiannotator()` (singular to match class name)

**Review by justinsalamon — APPROVED** [review]
👍

**Pull request merged**